src/main/java/pro/quicksense/modules/util/EmailUtil.java

```diff
@@ -0,0 +1,74 @@
+package pro.quicksense.modules.util;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.data.redis.core.RedisTemplate;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.stereotype.Service;
+import pro.quicksense.modules.common.CommonConstant;
+import pro.quicksense.modules.service.EmailService;
+
+import java.util.concurrent.TimeUnit;
+
+@Service
+@Slf4j
+public class EmailUtil implements EmailService {
```

Comment:
```suggestion
public class EmailServiceImpl implements EmailService {
```
rename `EmailUtil` to `EmailServiceImpl`, and move the file under `impl` folder to make it consistent with `UserServiceImpl`.